front/tests/pages/rollingstock-editor-page-model.ts

```diff
@@ -176,6 +212,30 @@ class RollingstockEditorPage extends CommonPage {
     expect(await this.loadingGauge.inputValue()).toBe(value);
   }
 
+  // Select primary category
+  async selectPrimaryCategory(value: RollingStockCategory) {
+    await this.primaryCategorySelector.selectOption(value);
+    await expect(this.primaryCategorySelector).toHaveValue(value);
+    await expect(this.categoriesCheckboxes[value]).toBeChecked();
+    await expect(this.categoriesCheckboxes[value]).toBeDisabled();
+  }
+
+  // Select an unselected category checkbox
+  async selectUnselectedCategory(category: RollingStockCategory) {
```

Comment:
somewhat redundant
 
```suggestion
  async selectCategory(category: RollingStockCategory) {
```
or 
```suggestion
  async checkCategoryCheckbox(category: RollingStockCategory) {
```

Reply:
Sure! I wanted to make it clear that trying to select an already selected category would cause test failure, but perhaps that should be implicit.

Reply:
Done ^^